pkg/kube/cluster-init.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/sh
 #
-# Copyright (c) 2023 Zededa, Inc.
+# Copyright (c) 2024 Zededa, Inc.
```

Comment:
Makes sense to add years and not remove i.e., 2023-2024

Reply:
I'm now looking at this wondering why I bumped it, I'll fix it and drink some coffee.